exercises/protein-translation/canonical-data.json

```diff
@@ -3,12 +3,12 @@
   "cases": [
     {
       "uuid": "2c44f7bf-ba20-43f7-a3bf-f2219c0c3f98",
-      "description": "Empty RNA sequence results in no proteins",
+      "description": "Empty RNA sequence results in no amino acids",
       "comments": [
         "Tracks may choose to expect an error when an empty RNA sequence  ",
         "is given, or use this test to expect an empty result.            "
       ],
-      "property": "proteins",
+      "property": "protein",
```

Comment:
The function returns a _single_ protein, not multiple ones.
An alternative name here would be `polypeptide`, but that would maybe require changing the whole exercise to be focused on polypeptide translation, so I left it as-is.

Reply:
The [CI is failing due to test-related changes.](https://github.com/exercism/problem-specifications/actions/runs/10979487489/job/30483946760?pr=2478) Should changes to the canonical data be done as part of a separate PR or not done at all?

Reply:
As the [readme](https://github.com/exercism/problem-specifications?tab=readme-ov-file#test-data-canonical-datajson) says, tests that are already written in `canonical-data`  are immutable, meaning you'd have to generate new tests with new UUIDs if you want to change the `property` field.

Reply:
Significant changes to the canonical data should first be discussed on the forum.